presto-main/src/main/java/io/prestosql/operator/PagesIndex.java

```diff
@@ -493,20 +493,15 @@ public LookupSourceSupplier createLookupSourceSupplier(
             // This code path will trigger only for OUTER joins. To fix that we need to add support for
             //        OUTER joins into NestedLoopsJoin and remove "type == INNER" condition in LocalExecutionPlanner.visitJoin()
 
-            try {
-                LookupSourceSupplierFactory lookupSourceFactory = joinCompiler.compileLookupSourceFactory(types, joinChannels, sortChannel, outputChannels);
-                return lookupSourceFactory.createLookupSourceSupplier(
-                        session,
-                        valueAddresses,
-                        channels,
-                        hashChannel,
-                        filterFunctionFactory,
-                        sortChannel,
-                        searchFunctionFactories);
-            }
-            catch (Exception e) {
-                log.error(e, "Lookup source compile failed for types=%s error=%s", types, e);
-            }
+            LookupSourceSupplierFactory lookupSourceFactory = joinCompiler.compileLookupSourceFactory(types, joinChannels, sortChannel, outputChannels);
+            return lookupSourceFactory.createLookupSourceSupplier(
+                    session,
+                    valueAddresses,
+                    channels,
+                    hashChannel,
+                    filterFunctionFactory,
+                    sortChannel,
+                    searchFunctionFactories);
         }
 
         // if compilation fails
```

Comment:
update this too

Reply:
addressed in https://github.com/trinodb/trino/pull/10033. thanks @sopel39 for taking care of this :)